### DISCUSSION TITLE
nRF5340 (nRF53 series) support?

### DISCUSSION THREAD

**Discussion by Hedda** [ffenix113/zigbee_home#20] [General]
nRF5340 is the latest SoC from Nordic Semiconductor with Zigbee support (combining all features of nRF52840 and nRF52833)

https://www.nordicsemi.com/products/nrf5340

It features 1 MB Flash + 512 kB RAM, and it provides better RX sensitivity (less sensitive to EMF/RMI noise) while using less power.

https://www.nordicsemi.com/-/media/Software-and-other-downloads/Product-Briefs/nRF5340-SoC-PB.pdf

https://blog.nordicsemi.com/getconnected/why-does-the-nrf5340-have-two-cores

  "**_The 0 dBm TX current is 3.4 mA, while the RX current is only 2.7 mA, resulting in a reduction of 29% and 41% when comparing to the nRF52840 SoC. The RX sensitivity is -98 dBm, 3 dB better than nRF52840, meaning that the nRF5340 provides 3 dB better sensitivity, while using 41% less current._**"

https://infocenter.nordicsemi.com/index.jsp?topic=%2Fps_nrf5340%2Fkeyfeatures_html5.html

https://www.nordicsemi.com/Products/Wireless/Zigbee

https://www.nordicsemi.com/Products/Development-software/nrf5-sdk-for-thread-and-zigbee

Other than using about half the power, another reason why you might want to add support for it is for its nRF5340-DK development kit:

https://www.nordicsemi.com/Products/Development-hardware/nrf5340-dk

Ebyte/cdEbyte have an E83-2G4M03S SMD radio module as well which can be found on their E83-2G4M03S-TB evaluation board

https://ebyteiot.com/products/nrf5340-wireless-mesh-bluetooth-test-board-usb-interface-cdebyte-e83-2g4m03s-tb-easy-to-develop-bluetooth-test-kit-ipex-antenna

https://www.cdebyte.com/products/E83-2G4M03S

https://www.cdebyte.com/products/E83-2G4M03S-TB

https://www.amazon.co.uk/EBYTE-nRF5340-Wireless-Bluetooth-E83-2G4M03S-TB/dp/B0CKQGX9M2

https://www.aliexpress.com/item/1005006108259017.html

https://www.alibaba.com/product-detail/EBYTE-E83-2G4M03S-TB-Small-size_1600959383696.html

https://www.alibaba.com/product-detail/COJXU-E83-2G4M03S-Small-size-and_1600959538223.html

![image](https://github.com/ffenix113/zigbee_home/assets/6320001/70f7375c-ff48-4d86-b380-0b980075ac10)

For commercial and DIY projects there is a nRF5340 based radio modules (BT40F, BT40, BT40E, BT40N) from Fanstel Corp.

https://www.fanstel.com/bt40f-nrf5340

https://www.fanstel.com/bc40m-nrf5340

and u-blox

https://www.fanstel.com/buy/bt840f-v1-nrf52840-bluetooth-5-thread-zigbee-module-hscxm-n7trh

https://www.u-blox.com/en/search?query=nRF5340

**Comment by lorenzspenger** [comment]
> Initial goal of the project is to support nRF52840 based devices, with expansion to nRF53 series.

from the readme
